lib/nodekey/NodeKey.ts

```diff
@@ -112,6 +114,19 @@ class NodeKey {
     }
     await fs.writeFile(path, buf);
   }
+
+  /**
+   * Derives a child seed from the private key for the swap client
+   * @param swapClient the swap client to create the seed for
+   */
+  public childSeed = (swapClient: SwapClientType) => {
+    const privKeyHex = this.privKey.toString('hex');
+    const privKeyHexHash = createHash('sha256')
+      .update(`${privKeyHex}-${swapClient}`)
+      .digest('hex');
```

Comment:
Is there any need to digest to hex? I think we could just do `digest()` and pass a `Buffer` to `entropyToMnemonic` below.